docs/API_SPECS_SIMULATIONS.md

```diff
@@ -8,8 +8,8 @@ The service supports only one simulation, with ID "1".
 ### Creation with POST
 
 When invoking the API using the POST HTTP method, the service will always
-attempt to create a new simulation. Since the service allows
-to create only one simulation, retrying the request will result in errors
+attempt to create a new simulation. The service allows
+to create multiple simulation but there can only be one running simulation. Retrying the request to create a new running simulation will result in errors
 after the first request has been successfully completed.
```

Comment:
I think we need to clarify this part. If it is possible to create multiple simulations, but there can be only one running, when will POST allow to create a second simulation and when not? Does the service check if there is already a simulation running?

Reply:
please also split long lines, around the 76th column